BraveWallet/Crypto/CryptoView.swift

```diff
@@ -105,6 +106,65 @@ public struct CryptoView: View {
               )
             case .panelUnlockOrSetup:
               EmptyView()
+            case .accountSelection:
+              AccountListView(keyringStore: keyringStore)
+            case .transactionHistory:
+              AccountTransactionListView(
+                keyringStore: walletStore.keyringStore,
+                activityStore: store.accountActivityStore(for: walletStore.keyringStore.selectedAccount),
+                networkStore: store.networkStore
+              )
+            case .buySendSwap(let destination):
+              switch destination.kind {
+              case .buy:
+                BuyTokenView(
+                  keyringStore: keyringStore,
+                  networkStore: store.networkStore,
+                  buyTokenStore: store.openBuyTokenStore(destination.initialToken),
+                  onDismiss: {
+                    onBSSShortcutDismiss?()
```

Comment:
I think it might be a bit cleaner to just use an `onDisappear` closure on these `BuyTokenView` / `SendTokenView` / `SwapCryptoView` instead of adding this new onDismiss closure (could even wrap them in a `Group` so only one `onDisappear` is used). I think we can just call `store.closeBSSStores()` directly too without needing this `onBSSShortcutDismiss` closure on `CryptoView`.

Reply:
That will require to pass entire `CryptoStore` to each the view. Basically this `onDismiss` is only for users that comes from wallet panel directly. I will change the closure to just call `store.closeBSSStores()`
For regular flow we will keep the same to rely on `buySendSwapDestination` didSet